features/api/services/application_plans/new.feature

```diff
@@ -0,0 +1,52 @@
+@javascript
+Feature: Product's new application plans page
+
+  Background:
+    Given a provider is logged in
+    And a product
+
+  Scenario: Navigation
+    Given they go to the product's application plans admin page
+    When they select toolbar action "Create application plan"
+    Then the current page is the product's new application plan admin page
+
+  @onpremises
+  Scenario: Create an application plan when onpremises
+    Given they go to the product's new application plan admin page
+    When the form is submitted with:
+      | Name        | Premium |
+      | System name | premium |
+    Then the current page is the product's application plans admin page
+    And they should see "Created application plan Premium"
+    And the table has the following row:
+      | Name    | Contracts | State  |
+      | Premium | 0         | hidden |
+
+  Scenario: Create an application plan
```

Comment:
Hm, what's the difference between this one and the previous scenario?

Reply:
It's origins date back to this integration test: https://github.com/3scale/system/commit/4f8c27c73aaeb028781f90d5629f2e7cb6786450#diff-6686769bc74a2494510c9a34f6368fbd47327e1ff8d6437f43bacde44ff93581R108-R115
When implementing this page with React, the test was transformed into a cucumber but it's not clear anymore what it should test. I will update so that it tests the same as before.